web/src/people.js

```diff
@@ -232,6 +232,18 @@ export function user_ids_string_to_ids_array(user_ids_string) {
     return ids;
 }
 
+export function get_participants_from_user_ids_string(user_ids_string) {
+    const user_ids = user_ids_string_to_ids_array(user_ids_string);
+    if (!(user_ids.length === 1 && user_ids[0] === my_user_id)) {
+        // For group PMs or 1:1 private messages, the user_ids_string
+        // contains just the other user, so we need to add ourselves if not
+        // already present. For PM to self, the current user is already present,
+        // in user_ids_string, so we don't need to add it.
+        user_ids.push(my_user_id);
+    }
+    return new Set(user_ids);
```

Comment:
I think we get to delete the if statement and tweak the comment with  the `new Set` design, right? That was my goal -- getting rid of the hard to read conditional for "when the user_id is already there".

Reply:
Okay, updated!